tests/websockets/src/jvmTest/kotlin/CloudRouterTest.kt

```diff
@@ -0,0 +1,33 @@
+
+import cloud.router.GetReviewSubscription
+import com.apollographql.apollo3.ApolloClient
+import com.apollographql.apollo3.network.http.HttpNetworkTransport
+import com.apollographql.apollo3.network.http.LoggingInterceptor
+import com.apollographql.apollo3.testing.internal.runTest
+import fullstack.tutorial.TripsBookedSubscription
+import kotlin.test.Ignore
+import kotlin.test.Test
+
+// Ignored because it depends on a remote server
+@Ignore
+class CloudRouterTest {
+  @Test
+  fun simple() = runTest {
+    val apolloClient = ApolloClient.Builder()
+        .serverUrl("")
+        .subscriptionNetworkTransport(
+            HttpNetworkTransport.Builder()
+                .serverUrl("https://main--subs-testing.staging.cloud.starstuff.dev/graphql")
```

Comment:
I remember Hugh saying that this staging endpoint wouldn't stick around so if this is a CI job it'll start failing at some point.

Reply:
It's mostly as a reference for my future self. This test is ignored (see the `@Ignore` annotation on top of this class). I like keeping them around cause it's handy to get back to them, uncomment the `@Ignore` and try stuff. But they're not part of any CI test.